docs/user/dashboard/dashboard.asciidoc

```diff
@@ -116,56 +158,99 @@ To compare the data in the panels, arrange the panels on the dashboard, or remov
-  get prompted with field names and operators you can use to build a structured query. 
-  For example, in the sample web logs data, this query displays data only for the US:
+Combine free text search with field-based search using the <<kuery-query,{kib} Query Language>>.
+Type a search term to match across all fields, or begin typing a field name to
+get prompted with field names and operators you can use to build a structured query. 
 
-  . Enter `g`, then select *geo.source*.
-  . Select *equals some value* and *US*, then click *Update*.
-  . For a more complex search, try:
+For example, in the sample web logs data, the following query displays data only for the US:
+
+. Enter `g`, then select *geo.source*.
+
+. Select *equals some value* and *US*, then click *Update*.
+
+. For a more complex search, try:
 
 [source,text]
 -------------------
 geo.src : "US" and url.keyword : "https://www.elastic.co/downloads/beats/metricbeat"
 -------------------
 
-Time filter::
-  Dashboards have a global time filter that restricts the data that displays, but individual panels can
-  override the global time filter.
+[float]
+[[time-filter]]
+==== Time filter
+
+The <<set-time-filter,time filter>> restrict the data that appears on the dashboard, but you can override the time filter with panel filters.
   
-  . To update the time filter, add a panel that displays time on the x-axis.
+. To update the time filter, add a panel that displays time-based data along the x-axis.
 
-  . Open the panel menu, then select *More > Customize time range*.
+. Open the panel menu, then select *More > Customize time range*.
 
-  . On the *Customize panel time range* window, specify the new time range, then click *Add to panel*.
-  
+. On the *Customize panel time range* window, specify the time range, then click *Add to panel*.
++
 [role="screenshot"]
 image:images/time_range_per_panel.gif[Time range per dashboard panel]
 
-Additional filters with AND::
-  Add filters to a dashboard, or pin filters to multiple places in {kib}. To add filters, using a basic editor or an advanced JSON editor for the {es} {ref}/query-dsl.html[query DSL]. 
-  When you use more than one index pattern on a dashboard, the filter editor allows you to filter only one dashboard.
-  To dynamically add filters, click a series on a dashboard. For example, to filter the dashboard to display only ios data:
-  . Click *Add filter*.
-  . Set *Field* to *machine.os*, *Operator* to *is*, and *Value* to *ios*.
-  . *Save* the filter.
-  . To remove the filter, click *x*.
+[float]
+[[additional-filters-with-and]]
+==== Additional filters with AND
+
+Add filters to a dashboard, or pin filters to multiple places in {kib}. To add filters, use a basic editor or an advanced JSON editor for the {es} {ref}/query-dsl.html[query DSL]. 
+When you use more than one index pattern on a dashboard, the filter editor allows you to filter only one dashboard.
```

Comment:
I don't quite understand this line. Maybe something like this would better explain what happens when you are using more than one index pattern?

> If you use more than one index pattern on a dashboard, the filter editor allows you to select which index pattern to choose fields from.